### PR TITLE
placement: improve prop-types

### PR DESCRIPTION
**Pull request by macabeus**
## Context
Improve `prop-types` restricting the amount of values accepted by `placement` on `Tooltip` and `Popover` components.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f86a42e5c6690d0a5af3892444360e71605c8512
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f86a42e5c6690d0a5af3892444360e71605c8512
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by macabeus** [comment]
**git-cop** maldito. To com sono, depois vejo isso.

File: src/Popover/Popover.js [hunk 1]
```diff
@@ -131,7 +131,20 @@ Popover.propTypes = {
...
+    'topStart',
+    'bottomStart',
+    'bottomCenter',
+    'bottomEnd',
+    'topCenter',
+    'leftStart',
+    'leftMiddle',
+    'leftEnd',
+    'topEnd',
+  ]),
```
Comment (uselessdev) [inline src/Popover/Popover.js]: ❤️

File: src/Tooltip/index.js [hunk 2]
```diff
@@ -129,7 +129,20 @@ Tooltip.propTypes = {
...
+    'topStart',
+    'bottomStart',
+    'bottomCenter',
+    'bottomEnd',
+    'topCenter',
+    'leftStart',
+    'leftMiddle',
+    'leftEnd',
+    'topEnd',
+  ]),
```
Comment (uselessdev) [inline src/Tooltip/index.js]: ❤️

**Comment by macabeus** [comment]
@Deivis @miglsoares @uselessdev done

**Pull request merged**